pkg/info/info.go

```diff
@@ -36,19 +44,23 @@ func getEksctlVersion() string {
 
 // getKubectlVersion returns the kubectl version
 func getKubectlVersion() string {
-	cmd := exec.Command("kubectl", "version", "--short", "--client")
+	cmd := exec.Command("kubectl", "version", "--client", "--output", "json")
```

Comment:
Note the command `kubectl version --client --short` will generate STDERR for `v1.24.0`.

```
Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
```